src/modules/displayData.js

```diff
@@ -0,0 +1,17 @@
+const displayData = (data) => {
+  const listItems = document.querySelector('.list-group');
+  const { result } = data;
+
+  listItems.innerHTML = '';
+  result.forEach((item) => {
+    listItems.innerHTML += `<li class="list-group-item">${item.user}: <span class="floatRight badge badge-primary badge-pill">${item.score}</span></li>`;
+  });
+
+//   list.innerHTML = '';
+//   scores.forEach((score) => {
+//     list.innerHTML += `<li>${score[0]}: ${score[1]}</li>`;
+//   });
+//   if (scores.length > 5) { list.classList.add('scroll'); }
```

Comment:
- [ ] Kindly remove this commented code. It makes the project quite untidy. Refer to GitHistory if you want to reuse previous code.

Reply:
Thanks a lot for the heads-up! Changed applied 😎